src/ComboboxInput/__snapshots__/ComboboxInput.test.js.snap

```diff
@@ -4,74 +4,82 @@ exports[`<ComboboxInput /> create combobox input 1`] = `
 <div
   className="fd-combobox-input"
 >
-  <span
-    onFocus={[Function]}
-    onMouseDown={[Function]}
-    onTouchStart={[Function]}
+  <div
+    className="fd-popover"
   >
-    <div
-      className="fd-popper__control fd-popover"
+    <span
```

Comment:
i know this is in the snap shot, but it is generally not good practice to put `<div>`s inside of `<span>`s. 

This is coming from the clickoutside library. Is emiting `<div>` vs `<span>` configurable? If not, should it be?

Reply:
It is currently not configurable, but I just created an (enhancement) issue in that repo to allow for that customization.

Reply:
that was... fast:

https://github.com/nanot1m/react-foco/pull/3